src/hierarchies/updaters/mnig_updater.h

```diff
@@ -5,6 +5,15 @@
 #include "src/hierarchies/likelihoods/uni_lin_reg_likelihood.h"
 #include "src/hierarchies/priors/mnig_prior_model.h"
 
+//! Updater specific for the `UniLinRegLikelihood` used in combination
+//! with `MNIGPriorModel`, that is the model
```

Comment:
We need to make the notation coherent between `UniLinRegLikelihood`, `MNIGPriorModel` and `MNIGUpdater`. In some cases, we use (`reg_coeffs`,`var`) as parameters, here we use (`beta`,`sigsq`). Not a big issue, but a simple suggestion.